docs/tutorials/onnx2tensorrt.md

```diff
@@ -69,18 +69,19 @@ python tools/deployment/onnx2tensorrt.py \
 
 The table below lists the models that are guaranteed to be convertable to TensorRT.
 
-|    Model    |                        Config                        | Status |
-| :---------: | :--------------------------------------------------: | :----: |
-|     SSD     |             `configs/ssd/ssd300_coco.py`             |   Y    |
-|    FSAF     |        `configs/fsaf/fsaf_r50_fpn_1x_coco.py`        |   Y    |
-|    FCOS     |   `configs/fcos/fcos_r50_caffe_fpn_4x4_1x_coco.py`   |   Y    |
-|   YOLOv3    |  `configs/yolo/yolov3_d53_mstrain-608_273e_coco.py`  |   Y    |
-|  RetinaNet  |   `configs/retinanet/retinanet_r50_fpn_1x_coco.py`   |   Y    |
-| Faster-RCNN | `configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py` |   Y    |
+|    Model    |                        Config                        | Dynamic Shape | batch inference | Note |
```

Comment:
Current we have not fully test dynamic shape and muti-batch features on onnx2tensorrt.